docs/docker-swarm.md

```diff
@@ -142,6 +142,7 @@ services:
   erpnext-nginx:
     image: frappe/erpnext-nginx:${ERPNEXT_VERSION?Variable ERPNEXT_VERSION not set}
     environment:
+      - UPSTREAM_REAL_IP_ADDRESS=10.0.0.0/8
```

Comment:
UPSTREAM_REAL_IP_RECURSIVE is off by default? does it need to be on here?

Reply:
This is only needed when you have more than one proxy server before nginx. Since there is only traefik, it can be left as off.